### DISCUSSION TITLE
Adaptive Card Design

### DISCUSSION THREAD

**Discussion by npc137** [microsoft/AdaptiveCards#8586] [Q&A]
Hi all,

I am trying to create an adaptive card that relays information from a submitted Dataverse form. Could anyone point me towards any documentation that could point me in the right direction on this?

Thanks!

**Comment by dominikake** [comment]
Is this what you're trying to do? Create a card with data from the dataverse? https://learn.microsoft.com/en-us/power-apps/cards/tutorials/dataverse-card
